.env.example

```diff
@@ -32,7 +32,7 @@ SCR_DEPLOY_CONFIG=${SCR_MONOREPO_DIR}/packages/contracts-bedrock/deploy-config/g
 # This is the commit in the https://github.com/ethereum-optimism/optimism/ repo
 # at which the chain's genesis was created. This is necessary to have our validation checks
 # recreate the genesis file using identical source code
-SCR_GENESIS_CREATION_COMMIT=""
+SCR_GENESIS_CREATION_COMMIT="33f06d2d5e4034125df02264a5ffe84571bd0359" # commit for tag op-contracts/v1.6.0
```

Comment:
Should we hint that this is only the default value, and it is possible/ok to override it?